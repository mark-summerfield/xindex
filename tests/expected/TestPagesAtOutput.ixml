<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE indexdata [
<!ELEMENT indexdata (source, fonts, records) >
<!ELEMENT source EMPTY >
<!ATTLIST source
    creator CDATA #REQUIRED
    version CDATA #REQUIRED
    time CDATA #REQUIRED >
<!-- time value is UTC in this format: 2011-03-03T03:41:14 -->
<!ELEMENT fonts (font+) >
<!ELEMENT font (fname, aname) >
<!ATTLIST font
    id CDATA #REQUIRED >
<!ELEMENT fname (#PCDATA) >
<!ELEMENT aname (#PCDATA) >
<!ELEMENT records (record)* >
<!ATTLIST records
    type CDATA #IMPLIED >
<!ELEMENT record (field+) >
<!ATTLIST record
    time CDATA #REQUIRED
    user CDATA #IMPLIED
    label CDATA #IMPLIED
    deleted (y | n) #IMPLIED
    type CDATA #IMPLIED >
<!-- time value is UTC in this format: 2008-08-02T16:27:44 -->
<!-- label value is integer -->
<!-- type value can be "generated" (automatically generated) -->
<!ELEMENT field (#PCDATA | text | esc | literal | hide | sort)* >
<!ATTLIST field
    class CDATA #IMPLIED >
<!-- class value can be "locator" -->
<!ELEMENT text EMPTY >
<!ATTLIST text
    font CDATA #IMPLIED
    color CDATA #IMPLIED
    smallcaps ( y | n ) #IMPLIED
    style ( b | i | u | bi | bu | iu | biu ) #IMPLIED
    offset ( u | d ) #IMPLIED
>
<!-- font and color attribute values are integers in range 0-31 -->
<!ELEMENT literal EMPTY >
<!-- literal: forces the succeeding character to be used in sort -->
<!ELEMENT hide (#PCDATA) >
<!-- hide: contains text to be ignored in sorting -->
<!ELEMENT sort (#PCDATA) >
<!-- sort: contains text to be used in sorting but not displayed -->
]>
<indexdata>
<source creator="XindeX" version="0.8.0" time="2016-01-21T12:59:47"/>
<fonts>
<font id="0">
    <fname>Arial</fname>
    <aname>Arial</aname>
</font>
<font id="1">
    <fname>Courier New</fname>
    <aname>Courier New</aname>
</font>
</fonts>
<records>
<record time="2016-01-21T12:59:47">
    <field>One</field>
    <field class="locator">1</field>
</record>
<record time="2016-01-21T12:59:47">
    <field>One</field>
    <field class="locator">2</field>
</record>
<record time="2016-01-21T12:59:47">
    <field>One</field>
    <field class="locator"><text style="b"/>3<text/></field>
</record>
<record time="2016-01-21T12:59:47">
    <field>One</field>
    <field class="locator"><text style="i"/>15<text/><text style="i"/>–<text/><text style="i"/>20<text/></field>
</record>
<record time="2016-01-21T12:59:47">
    <field>Two</field>
    <field class="locator"><text style="i"/>15<text/><text style="i"/>–<text/><text style="i"/>20<text/></field>
</record>
<record time="2016-01-21T12:59:47">
    <field>Two</field>
    <field class="locator">17<text style="i"/>tbl<text/></field>
</record>
</records>
</indexdata>
