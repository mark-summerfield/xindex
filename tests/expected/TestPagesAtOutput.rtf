{\rtf1\ansi\deff0
{\fonttbl
    {\f0 Times New Roman;}
    {\f1 Arial;}
    {\f2 Courier New;}
}
{\header\pard\qr\plain\f0\chpgn\par}
{\info{\title XindeX 0.8.0}{\author Tester}}
{\pard\qc Index\par}
{\pard\par}
{{\pard\par}}{\pard\b\f1\fs22
\f1\fs22 \b O\b0 \f0\fs20 \par}
{{\pard\par}}{\pard\f0\fs20\fi-1000\li1000 One, 1, 2, \b 3\b0 , \i 15\endash 20\i0 \par}
{{\pard\par}}{\pard\b\f1\fs22
\f1\fs22 \b T\b0 \f0\fs20 \par}
{{\pard\par}}{\pard\f0\fs20\fi-1000\li1000 Two, 17\i tbl\i0 , \i 15\endash 20\i0 \par}
}
