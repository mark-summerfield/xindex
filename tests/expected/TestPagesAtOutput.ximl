<?xml version="1.0" encoding="UTF-8"?>
<XindeX version="100">
<entries>
<entry eid="1" peid="0" created="2016-01-21 12:59:47" updated="2016-01-21 12:59:47" gids="">
    <term>One</term>
    <sortas saf="A">one</sortas>
    <pages>1, 2, &lt;b&gt;3&lt;/b&gt;, &lt;i&gt;15–20&lt;/i&gt;</pages>
</entry>
<entry eid="2" peid="0" created="2016-01-21 12:59:47" updated="2016-01-21 12:59:47" gids="">
    <term>Two</term>
    <sortas saf="A">two</sortas>
    <pages>17&lt;i&gt;tbl&lt;/i&gt;, &lt;i&gt;15–20&lt;/i&gt;</pages>
</entry>
</entries>
<ignored_firsts>
    <ignore_firsts word="a"/>
    <ignore_firsts word="abaft"/>
    <ignore_firsts word="abeam"/>
    <ignore_firsts word="aboard"/>
    <ignore_firsts word="about"/>
    <ignore_firsts word="above"/>
    <ignore_firsts word="absent"/>
    <ignore_firsts word="according to"/>
    <ignore_firsts word="across"/>
    <ignore_firsts word="afore"/>
    <ignore_firsts word="after"/>
    <ignore_firsts word="against"/>
    <ignore_firsts word="ahead of"/>
    <ignore_firsts word="along"/>
    <ignore_firsts word="alongside"/>
    <ignore_firsts word="amid"/>
    <ignore_firsts word="amidst"/>
    <ignore_firsts word="among"/>
    <ignore_firsts word="amongst"/>
    <ignore_firsts word="an"/>
    <ignore_firsts word="and"/>
    <ignore_firsts word="apart from"/>
    <ignore_firsts word="apropos"/>
    <ignore_firsts word="around"/>
    <ignore_firsts word="as"/>
    <ignore_firsts word="as far as"/>
    <ignore_firsts word="as for"/>
    <ignore_firsts word="aside"/>
    <ignore_firsts word="aside from"/>
    <ignore_firsts word="as long as"/>
    <ignore_firsts word="as of"/>
    <ignore_firsts word="as opposed to"/>
    <ignore_firsts word="as per"/>
    <ignore_firsts word="as regards"/>
    <ignore_firsts word="as soon as"/>
    <ignore_firsts word="astern of"/>
    <ignore_firsts word="astride"/>
    <ignore_firsts word="as well as"/>
    <ignore_firsts word="at"/>
    <ignore_firsts word="athwart"/>
    <ignore_firsts word="atop"/>
    <ignore_firsts word="back to"/>
    <ignore_firsts word="barring"/>
    <ignore_firsts word="because of"/>
    <ignore_firsts word="before"/>
    <ignore_firsts word="behind"/>
    <ignore_firsts word="below"/>
    <ignore_firsts word="beneath"/>
    <ignore_firsts word="beside"/>
    <ignore_firsts word="besides"/>
    <ignore_firsts word="between"/>
    <ignore_firsts word="beyond"/>
    <ignore_firsts word="but"/>
    <ignore_firsts word="by"/>
    <ignore_firsts word="circa"/>
    <ignore_firsts word="close to"/>
    <ignore_firsts word="concerning"/>
    <ignore_firsts word="despite"/>
    <ignore_firsts word="down"/>
    <ignore_firsts word="due to"/>
    <ignore_firsts word="during"/>
    <ignore_firsts word="except"/>
    <ignore_firsts word="except for"/>
    <ignore_firsts word="excluding"/>
    <ignore_firsts word="failing"/>
    <ignore_firsts word="far from"/>
    <ignore_firsts word="following"/>
    <ignore_firsts word="for"/>
    <ignore_firsts word="from"/>
    <ignore_firsts word="given"/>
    <ignore_firsts word="in"/>
    <ignore_firsts word="including"/>
    <ignore_firsts word="inside"/>
    <ignore_firsts word="inside of"/>
    <ignore_firsts word="instead of"/>
    <ignore_firsts word="in to"/>
    <ignore_firsts word="into"/>
    <ignore_firsts word="left of"/>
    <ignore_firsts word="like"/>
    <ignore_firsts word="mid"/>
    <ignore_firsts word="midst"/>
    <ignore_firsts word="minus"/>
    <ignore_firsts word="modulo"/>
    <ignore_firsts word="near"/>
    <ignore_firsts word="near to"/>
    <ignore_firsts word="next"/>
    <ignore_firsts word="next to"/>
    <ignore_firsts word="nor"/>
    <ignore_firsts word="notwithstanding"/>
    <ignore_firsts word="of"/>
    <ignore_firsts word="off"/>
    <ignore_firsts word="on"/>
    <ignore_firsts word="on to"/>
    <ignore_firsts word="onto"/>
    <ignore_firsts word="opposite"/>
    <ignore_firsts word="opposite of"/>
    <ignore_firsts word="opposite to"/>
    <ignore_firsts word="or"/>
    <ignore_firsts word="out"/>
    <ignore_firsts word="out from"/>
    <ignore_firsts word="out of"/>
    <ignore_firsts word="outside"/>
    <ignore_firsts word="outside of"/>
    <ignore_firsts word="over"/>
    <ignore_firsts word="owing to"/>
    <ignore_firsts word="pace"/>
    <ignore_firsts word="past"/>
    <ignore_firsts word="per"/>
    <ignore_firsts word="plus"/>
    <ignore_firsts word="prior to"/>
    <ignore_firsts word="pro"/>
    <ignore_firsts word="pursuant to"/>
    <ignore_firsts word="qua"/>
    <ignore_firsts word="rather than"/>
    <ignore_firsts word="regarding"/>
    <ignore_firsts word="regardless of"/>
    <ignore_firsts word="right of"/>
    <ignore_firsts word="round"/>
    <ignore_firsts word="sans"/>
    <ignore_firsts word="save"/>
    <ignore_firsts word="since"/>
    <ignore_firsts word="so"/>
    <ignore_firsts word="subsequent to"/>
    <ignore_firsts word="such as"/>
    <ignore_firsts word="than"/>
    <ignore_firsts word="thanks to"/>
    <ignore_firsts word="that of"/>
    <ignore_firsts word="the"/>
    <ignore_firsts word="through"/>
    <ignore_firsts word="throughout"/>
    <ignore_firsts word="times"/>
    <ignore_firsts word="to"/>
    <ignore_firsts word="toward"/>
    <ignore_firsts word="towards"/>
    <ignore_firsts word="under"/>
    <ignore_firsts word="underneath"/>
    <ignore_firsts word="unlike"/>
    <ignore_firsts word="until"/>
    <ignore_firsts word="up"/>
    <ignore_firsts word="upon"/>
    <ignore_firsts word="up to"/>
    <ignore_firsts word="versus"/>
    <ignore_firsts word="via"/>
    <ignore_firsts word="vice"/>
    <ignore_firsts word="with"/>
    <ignore_firsts word="within"/>
    <ignore_firsts word="without"/>
    <ignore_firsts word="worth"/>
    <ignore_firsts word="yet"/>
</ignored_firsts>
<configs>
    <config key="AltFont">Arial</config>
    <config key="AltFontSize">9</config>
    <config key="Created">2016-01-21 12:59:47</config>
    <config key="Creator">Tester</config>
    <config key="GenericConjunction">●and●</config>
    <config key="HighestPageNumber">1999</config>
    <config key="IgnoreSubFirsts">1</config>
    <config key="Indent">0</config>
    <config key="Initials">T</config>
    <config key="Language">American</config>
    <config key="LargestPageRange">10</config>
    <config key="MonoFont">Courier New</config>
    <config key="MonoFontAsStrikeout">0</config>
    <config key="MonoFontSize">9</config>
    <config key="MostPages">10</config>
    <config key="Note"></config>
    <config key="PadDigits">4</config>
    <config key="PageRangeRules">pageRangeCMS16</config>
    <config key="RunInSeparator">;●</config>
    <config key="SectionPostLines">1</config>
    <config key="SectionPreLines">1</config>
    <config key="SectionSpecialTitle">&lt;span style=&quot;font-size: 11pt; font-family: &#x27;Arial&#x27;;&quot;&gt;&lt;b&gt;Symbols &amp;amp; Numbers&lt;/b&gt;&lt;/span&gt;</config>
    <config key="SectionTitles">1</config>
    <config key="See">&lt;i&gt;See&lt;/i&gt; </config>
    <config key="SeeAlso">&lt;i&gt;See also&lt;/i&gt; </config>
    <config key="SeeAlsoPosition">1</config>
    <config key="SeeAlsoPrefix">.●</config>
    <config key="SeeAlsoSeparator">;●</config>
    <config key="SeeAlsoSuffix"></config>
    <config key="SeePrefix">.●</config>
    <config key="SeeSeparator">;●</config>
    <config key="SeeSuffix"></config>
    <config key="SortAsRules">wordByWordCMS16</config>
    <config key="StdFont">Times New Roman</config>
    <config key="StdFontSize">10</config>
    <config key="Style">-1</config>
    <config key="SubSee">&lt;i&gt;see&lt;/i&gt; </config>
    <config key="SubSeeAlso">&lt;i&gt;see also&lt;/i&gt; </config>
    <config key="SubSeeAlsoPosition">1</config>
    <config key="SubSeeAlsoPrefix">●(</config>
    <config key="SubSeeAlsoSeparator">;●</config>
    <config key="SubSeeAlsoSuffix">)</config>
    <config key="SubSeePrefix">●(</config>
    <config key="SubSeeSeparator">;●</config>
    <config key="SubSeeSuffix">)</config>
    <config key="SuggestSpelled">1</config>
    <config key="TermPagesSeparator">,●</config>
    <config key="Title">Index</config>
    <config key="UUID">84317D2220B24E3498B33E1467D0622C</config>
    <config key="Updated">2016-01-21 12:59:47</config>
    <config key="XRefToSubentry">0</config>
</configs>
</XindeX>
