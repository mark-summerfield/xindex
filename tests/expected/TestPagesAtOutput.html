<html>
    <!-- Generated by XindeX 0.8.0 for Tester on 2016-01-21 -->
    <head>
        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
        <title>Index</title>
    </head>
    <style>
    p { font-family: "Times New Roman"; font-size: "10pt"; }
    h2 { font-family: "Arial"; font-size: "11pt"; }
    .hi0 { padding-left: 3em; text-indent: -3em; }
    .hi1 { padding-left: 5em; text-indent: -3em; }
    .hi2 { padding-left: 7em; text-indent: -3em; }
    .hi3 { padding-left: 9em; text-indent: -3em; }
    .hi4 { padding-left: 11em; text-indent: -3em; }
    .hi5 { padding-left: 13em; text-indent: -3em; }
    .hi6 { padding-left: 15em; text-indent: -3em; }
    .hi7 { padding-left: 17em; text-indent: -3em; }
    .hi8 { padding-left: 19em; text-indent: -3em; }
    .hi9 { padding-left: 21em; text-indent: -3em; }
    </style>
<body>
<h1>Index</h1>
<p></p><h2>O</h2>
<p></p><p class="hi0">One, 1, 2, <b>3</b>, <i>15–20</i></p>
<p></p><h2>T</h2>
<p></p><p class="hi0">Two, 17<i>tbl</i>, <i>15–20</i></p>
</body></html>